commonmark/src/main/java/org/commonmark/parser/Parser.java

```diff
@@ -48,6 +50,16 @@ public Node parse(String input) {
         }
         return document;
     }
+    
+    public Node parse(Reader input) throws IOException {
```

Comment:
Hm... From my point of view it's ths same logic.
Please share your thoughts why you think or prefer make it separate.